crypto/onetimesig.go

```diff
@@ -149,16 +149,23 @@ type OneTimeSignatureSecrets struct {
 type OneTimeSignatureSecretsPersistent struct {
 	_struct struct{} `codec:",omitempty,omitemptyarray"`
 
+	// Truncated is used to indicate that the results have been truncated beyond the normal
+	// used key removal process. This flag is used to generate an error if an attempt is made
+	// to serialize a truncated object.
+	Truncated bool `codec:"-"`
+
```

Comment:
`Truncated` need not be exported, right?

Reply:
I think you're right. Probably nothing in this struct should be exported.

Reply:
The rest probably need it for `codec` to serialize them.